src/components/reservation/ReservationModalBody.tsx

```diff
@@ -156,23 +142,17 @@ const ReservationModalBody = ({
     reservationResponse?.reservationResults[0]?.reservationDetails;
 
   return (
-    <Modal
-      modalId={reservationModalId(faustId)}
-      screenReaderModalDescriptionText={t(
-        "reservationModalScreenReaderModalDescriptionText"
-      )}
-      closeModalAriaLabelText={t("reservationModalCloseModalAriaLabelText")}
-    >
+    <>
       {!reservationResult && (
         <section className="reservation-modal">
           <header className="reservation-modal-header">
-            <Cover id={pid} size="medium" animate />
+            <Cover id={allPids[0]} size="medium" animate />
             <div className="reservation-modal-description">
               <div className="reservation-modal-tag">
-                {materialTypes[0].specific}
+                {selectedManifestations[0].materialTypes[0].specific}
               </div>
               <h2 className="text-header-h2 mt-22 mb-8">
-                {mainTitle}
+                {selectedManifestations[0].titles.main}
```

Comment:
Please refactor this to a single location where we get a representational manifestation from the array of manifestations.

A helper which is able to determine this based on the work would be nice but just getting the first entry is sufficient. *Then* we can extract the pid, type, title etc. from that manifestation without the direct array access and any uncertainty about there is a manifestation or not.